src/server/pkg/obj/google_client.go

```diff
@@ -16,13 +15,7 @@ type googleClient struct {
 	bucket *storage.BucketHandle
 }
 
-func newGoogleClient(bucket string, credFile string) (*googleClient, error) {
-	opts := []option.ClientOption{option.WithScopes(storage.ScopeFullControl)}
```

Comment:
I think the scope option can stay in this function because we are using it regardless. Just append it to the passed in options.